src/app/content/components/Toolbar/HighlightsPopUp.tsx

```diff
@@ -0,0 +1,441 @@
+        </ImagesGrid>
+      </GridWrapper>
+    </PopupBody>;
+  };
+
+  public myHighlights = () => {
+    return <PopupBody>
+      <GeneralLeftText>
+        <FormattedMessage id='i18n:toolbar:highlights:popup:heading:no-highlights'>
+          {(msg: Element | string) => msg}
+        </FormattedMessage>
+      </GeneralLeftText>
+      <MyHighlightsWrapper>
+        <GeneralText>
+          <FormattedMessage id='i18n:toolbar:highlights:popup:body:add-highlight'>
+            {(msg: Element | string) => msg}
+          </FormattedMessage>
+        </GeneralText>
+        <GeneralTextWrapper>
+          <FormattedMessage id='i18n:toolbar:highlights:popup:body:use-this-page'>
+            {(msg: Element | string) => msg}
+          </FormattedMessage>
+        </GeneralTextWrapper>
+        <MyHighlightsImage src={myHighlightsEmptyImage}></MyHighlightsImage>
+      </MyHighlightsWrapper>
+    </PopupBody>;
+  };
+
+  public blueNote = () => {
+    return ([
+    <BlueStickyNote>
+      <StickyNoteUl>
+        <StickyNoteLi>
+          <FormattedMessage id='i18n:toolbar:highlights:popup:body:note:highlight'>
+            {(msg: Element | string) => msg}
+          </FormattedMessage>
+        </StickyNoteLi>
+        <StickyNoteLi>
+          <FormattedMessage id='i18n:toolbar:highlights:popup:body:note:add-notes'>
+            {(msg: Element | string) => msg}
+          </FormattedMessage>
+        </StickyNoteLi>
+      </StickyNoteUl>
+    </BlueStickyNote>]);
+  };
+
+  public greenNote = () => {
+    return ([
+      <GreenStickyNote>
+        <StickyNoteUl>
+          <StickyNoteLi>
+            <FormattedMessage id='i18n:toolbar:highlights:popup:body:note:filter-chapters'>
+              {(msg: Element | string) => msg}
+            </FormattedMessage>
+            <InfoIconWrapper>
+              <InfoIcon/>
+              <Tooltip>
+                <FormattedMessage id='i18n:toolbar:highlights:popup:body:tooltip:review'>
+                  {(msg: Element | string) => msg}
+                </FormattedMessage>
+              </Tooltip>
+            </InfoIconWrapper>
+          </StickyNoteLi>
+          <StickyNoteLi>
+            <FormattedMessage id='i18n:toolbar:highlights:popup:body:note:study-guide'>
+              {(msg: Element | string) => msg}
+            </FormattedMessage>
+          </StickyNoteLi>
+        </StickyNoteUl>
+      </GreenStickyNote>,
+    ]);
+  };
+
+  public render() {
+    return (
+      this.props.myHighlightsOpen ?
+        <Modal>
+          <Mask>
+            <Wrapper>
+              <Header>
+                <FormattedMessage id='i18n:toolbar:highlights:popup:heading'>
+                  {(msg: Element | string) => msg}
+                </FormattedMessage>
+                <CloseIcon data-testid='close-highlights-popup' onClick={() => this.props.closeMyHighlights()}/>
+              </Header>
+              {this.props.user ? this.myHighlights() : this.loginForHighlights()}
```

Comment:
we should split the `myHighlights` and `loginForHighlights` content into separate files, this file is getting really big already.